src/components/Table/GridTable.tsx

```diff
@@ -1014,11 +1016,19 @@ const ObservedGridRow = React.memo((props: GridRowProps<any, any>) => (
 ));
 
 /** If a column def return just string text for a given row, apply some default styling. */
-function toContent(content: ReactNode | GridCellContent, isHeader: boolean, canSortColumn: boolean): ReactNode {
+function toContent(
+  content: ReactNode | GridCellContent,
+  isHeader: boolean,
+  canSortColumn: boolean,
+  style: GridStyle,
+): ReactNode {
   if (typeof content === "string" && isHeader && canSortColumn) {
     return <SortHeader content={content} />;
   } else if (isContentAndSettings(content)) {
     return content.content;
+  } else if (style.emptyCell && !content) {
```

Comment:
I wonder if we should support like a "export const noReallyEmpty" value that columns could use for cases that really do need empty? Seems like something that could come up?...

Reply:
Yea, probably a good idea. I was thinking that too, for instance there is sometimes content that we only want to show when the table is in `edit` mode, otherwise hide it in `read`.

Reply:
Do you think that'd be a property on the column, or would we want each cell to be able to define that?

Reply:
Yeah, I was thinking it'd be a return value from a cell, i.e. something like:

```
const secondColumn: ... = {
   data: emptyCell,
}
```

Where `emptyCell`-as-a-`const` means literally empty.

Reply:
Yea, that could work. What do you think of that vs something like:
```
const secondColumn = {
  data: "",
  allowEmpty: true,
}
```
Might be easier for implementers to discover the available prop vs finding the exported const helper. Or I guess they could even do:
```
const secondColumn = {
  data: <></>,
  // or
  data: " ",
}
```
That'd give the same result.

Reply:
Hm, yeah, the `GridColumn.allowEmpty` is across the whole column though, vs the `data` being per-cell. I guess we could do both...

I kinda like having just one, and since `data: emptyCell` is a superset of `allowEmpty` (i.e. you can do `data: emptyCell` a few times in a column to make the whole column effectively not empty, but you can't use `allowEmpty` to make only some of the cells empty), that's where I'm currently leaning.

But not against both/or start with one and see how it goes.

Reply:
Whoops, I definitely wrote that out wrong. I was actually thinking of it more like:
```
const secondColumn = { 
  data: () => { allowEmpty: true }
 }